src/main/java/com/basho/riak/client/core/query/timeseries/ColumnDescription.java

```diff
@@ -43,9 +38,7 @@ public ColumnType getType()
         INTEGER(1),
         FLOAT(2),
```

Comment:
This one is a little more interesting, since long is an even more loaded word. Are we referring to C's  32-bit `long`s, the 64-bit on 32-bit arch `long long`s, etc.  If it was SQL Server we would have `bigint`, Oracle would be `Numeric(someBigNum)`...